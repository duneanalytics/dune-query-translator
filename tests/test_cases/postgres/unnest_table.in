select unnest(c) from t