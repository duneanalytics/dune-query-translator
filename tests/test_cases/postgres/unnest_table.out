SELECT
  col
FROM t
CROSS JOIN UNNEST(c) AS _u(col)